### PR TITLE
Replace to babelify from 6to5ify

### PR DESCRIPTION
**Pull request by laiso**
https://github.com/babel/babelify

### CODE REVIEW

File: recipes/gulp.browserify/gulpfile.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 var gulp         = require('gulp');
 var gutil        = require('gulp-util');
 var source       = require('vinyl-source-stream');
-var sixtofiveify = require('6to5ify');
+var babelify     = require('babelify');
 var reactify     = require('reactify');
 var watchify     = require('watchify');
 var browserify   = require('browserify');
```
Comment (shinnn) [inline recipes/gulp.browserify/gulpfile.js]: Align `=`.

```js
var gulp        = require('gulp');
var gutil       = require('gulp-util');
var source      = require('vinyl-source-stream');
var babelify    = require('babelify');
var reactify    = require('reactify');
var watchify    = require('watchify');
var browserify  = require('browserify');
var browserSync = require('browser-sync');
```

**Comment by shinnn** [comment]
Good catch, @laiso!

Could you update the commit message to `Replace 6to5ify with babelify` after fixing https://github.com/BrowserSync/recipes/pull/7#discussion_r32884773?

**Comment by laiso** [comment]
Thanks for your advice. 
I think it's been fixed now.

**Comment by shinnn** [comment]
:100: Thanks, @laiso.

**Pull request merged**